applications/StructuralMechanicsApplication/custom_elements/total_lagrangian.cpp

```diff
@@ -154,16 +154,16 @@ namespace Kratos
         const GeometryType::IntegrationPointsArrayType& IntegrationPoints
         )
     {
-        const IntegrationMethod this_integration_method = GetGeometry().GetDefaultIntegrationMethod();
+        const IntegrationMethod this_integration_method = this->GetGeometry().GetDefaultIntegrationMethod();
         
         // Shape functions
-        rThisKinematicVariables.N = GetGeometry().ShapeFunctionsValues(rThisKinematicVariables.N, IntegrationPoints[PointNumber].Coordinates());
+        rThisKinematicVariables.N = this->GetGeometry().ShapeFunctionsValues(rThisKinematicVariables.N, IntegrationPoints[PointNumber].Coordinates());
```

Comment:
This is very slow. You should not pass the coordinates, only the is of the pointer! (I see the t it is not a change you did, still I see the error and it should be corrected)

Reply:
What?